benchmark/buffers/buffer-compare.js

```diff
@@ -8,8 +8,8 @@ var bench = common.createBenchmark(main, {
 function main(conf) {
   var iter = (conf.millions >>> 0) * 1e6;
   var size = (conf.size >>> 0);
-  var b0 = new Buffer(size).fill('a');
-  var b1 = new Buffer(size).fill('a');
+  var b0 = Buffer.allocUnsafe(size).fill('a');
+  var b1 = Buffer.allocUnsafe(size).fill('a');
```

Comment:
Shouldn't this use `Buffer.alloc(size[, fill[, encoding]])` instead of `.allocUnsafe(size).fill(fill)`?